Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -419,17 +419,17 @@ namespace OpenInfraPlatform
 				for (int i = 0; i < vec_cross_sections.size(); ++i)
 				{
 					std::shared_ptr<ProfileConverterT<IfcEntityTypesT>> profileConverter = profileCache->getProfileConverter(vec_cross_sections[i]);
-					const std::vector<std::vector<carve::geom::vector<2> > >& profile_coords = profileConverter->getCoordinates();
+					const std::vector<std::vector<carve::geom::vector<2> > >& profileCoords = profileConverter->getCoordinates();
 
 					// Save profile coords in paths
-					std::vector<std::vector<carve::geom::vector<2> > > profile_coords_2d;
-					for (int p = 0; p < profile_coords.size(); ++p)
+					std::vector<std::vector<carve::geom::vector<2> > > profileCoords2D;
+					for (int p = 0; p < profileCoords.size(); ++p)
 					{
-						const std::vector<carve::geom::vector<2> >& profile_loop = profile_coords[p];
+						const std::vector<carve::geom::vector<2> >& profileLoop = profileCoords[p];

```

Comment:
this could be simplified with

`for( const auto& profileLoop : profileCoords )`

Reply:
Resolved: 4c40ba7